README.md

```diff
@@ -1,8 +1,52 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nazanin Akbari . Jarvis Consulting
 
+My bachelor degree is in the field of Biomedical engineering. I have two years of work experience as a biomedical engineer in 5Azar hospital, Gorgan, Iran. I started my master degree at Concordia university, Montreal, QC in 2019. I got my master degree from Concordia university in the field of Electrical Engineering and Computer Science. At Concordia university, I did multiple projects in signal processing, image processing, Hardware verification and designing an Airline reservation system and I also designed some games. In Montreal, I also learned French language, B1 level. I like software industry because this field is incredibly broad. I am passionate to learn new things and expand my knowledge and experience. I love to solve problems and help people to experience a better and easier life.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Python, Google Cloud Platform
+
+**Competent:** Agile/Scrum, Git, Docker and Containerization, Intellij IDE, Visual Studio
+
+**Familiar:** Neural Network, Machine Learning, C#, Azure DevOps, Unit testing
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari)
+
+
+**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/creating-DDL/linux_sql)]:  Implemented Linux-SQL project, to help Jarvis Linux Cluster Administration (LCA) record the hardware specification of 10 servers running CentOS7 which are connected to each other throughout a switch. The project is implemented by using Bash scripts, Docker containers, Postgres database, and SQL queries.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/java-apps)]: Grep App: Implemented Java Grep application which is looking for regular expression that is searched in each file. This program is written in Java, using Streams, Lambda, and RegEx pattern matching, with Maven as the build tool. It is configured to be run as a command line with the JRE installed or via a Docker container.
```

Comment:
Implemented the Java Grep application looking for a regular expression searched in each file. This program is written in Java, using Streams, Lambda, and RegEx pattern matching, with Maven as the build tool. It is configured to be run as a command-line with the JRE installed or via a Docker container.